src/acp/mock/createAccessControlResourceDataset.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2021 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import type { SolidDataset } from "../../interfaces";
+import { addUrl, createSolidDataset, createThing, setThing } from "../..";
+import { acp } from "../../constants";
+import {
+  DEFAULT_ACCESS_CONTROL_RESOURCE_URL,
+  DEFAULT_DOMAIN,
+} from "./createAccessControlledResource";
+
+export const TEST_URL = {
+  accessControlResource: DEFAULT_ACCESS_CONTROL_RESOURCE_URL,
+  accessControl1: DEFAULT_DOMAIN.concat("ac1"),
+  memberAccessControl1: DEFAULT_DOMAIN.concat("mac1"),
+  accessControl1Policy1: DEFAULT_DOMAIN.concat("ac1p1"),
+  accessControl1AccessPolicy1: DEFAULT_DOMAIN.concat("ac1ap1"),
+  memberAccessControl1Policy1: DEFAULT_DOMAIN.concat("mac1p1"),
+  memberAccessControl1AccessPolicy1: DEFAULT_DOMAIN.concat("mac1ap1"),
+};
+
+/** @hidden */
+export function createAccessControlResourceDataset(): SolidDataset {
+  // Create dataset
```

Comment:
Hahaha, I was putting the comments in to get my head around the steps involved really. It did help me because when I scan the file, I'm going 1. Create dataset, 2. Create ACR...

All of this will be refactored and removed eventually.

I think I have a better version of mocking almost working. It will make the ACR structure much more evident and generic.